### ISSUE TITLE
Type error with children in Head component

### ISSUE THREAD

**Issue by sglass520** [inertiajs/inertia#1437]
### Version:

- `@inertiajs/react` version: 1.0.0

### Describe the problem:

According to documentation you should be able to have children in the Inertia's `Head` component. This is the only way to include a meta tag in the head.
```
import { Head } from '@inertiajs/react'

<Head>
  <title>Your page title</title>
  <meta name="description" content="Your page description" />
</Head>
```

However, this code results in the following error:
`Type '{ children: Element[]; }' has no properties in common with type 'IntrinsicAttributes & InertiaHeadProps'.`

Because the `Head` component only accepts `title` as props, there is no way to add meta to the head without a type error.

### Steps to reproduce:

1. Create TypeScript project using latest version of Inertia.
2. Paste code from documentation.

### PR
I believe this issue is in addition to the PR #1433 
Does the `InertiaHead` type need to be wrapped in `PropsWithChildren` generic?

**Comment by nogenem** [comment]
Just ran into this problem too... There is a bunch of other PRs and Issues about this and still nothing... ;/

**Comment by jessarcher** [comment]
Hey! I'm unable to replicate this in the React TypeScript playground at https://github.com/inertiajs/inertia/tree/master/playgrounds/react

Is there perhaps a `tsconfig.json` setting I need to enable?

I've merged #1433, but I don't _think_ that would have changed anything with this.

**Comment by sglass520** [comment]
> Hey! I'm unable to replicate this in the React TypeScript playground at https://github.com/inertiajs/inertia/tree/master/playgrounds/react
> 
> Is there perhaps a `tsconfig.json` setting I need to enable?
> 
> I've merged #1433, but I don't _think_ that would have changed anything with this.

Hey, thanks for checking on this. This is the `tsconfig.json` file in my project

```
{
    "compilerOptions": {
        "target": "es6",
        "lib": [
            "dom",
            "dom.iterable",
            "esnext"
        ],
        "jsx": "react-jsx",
        "module": "esnext",
        "moduleResolution": "node",
        "baseUrl": ".",
        "paths": {
            "@/*": ["resources/js/*"]
        },
        "types": [
            "vite/client"
        ],
        "isolatedModules": true,
        "allowSyntheticDefaultImports": true,
        "esModuleInterop": true ,
        "forceConsistentCasingInFileNames": true,
        "strict": true,
        "skipLibCheck": true
    }
}
```


v

**Comment by sglass520** [comment]
I created PR #1448 to explicitly declare children in type definition.

**Comment by jessarcher** [comment]
Thank you! I've merged #1448
